conans/client/cmd/profile.py

```diff
@@ -67,6 +67,8 @@ def cmd_profile_update(profile_name, key, value, cache_profiles_path):
         profile.options.update(tmp)
     elif first_key == "env":
         profile.env_values.update_replace(rest_key, value)
+    elif first_key == "conf":
+        profile.conf.update_conf_line("{}={}".format(rest_key, value))
```

Comment:
This is not what I meant, better do not convert it to string to have it parsed again inside the function.
Make a ``update_conf(rest_key, value)`` dedicated.